src/elements/Banner/Banner.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react"
+import styled from "styled-components"
+
+import { space } from "../../helpers"
+import { Box } from "../Box"
+import { Sans } from "../Typography"
+
+import { CloseIcon } from "../../svgs"
+
+const Target = styled.div`
+  height: 30px;
+  padding: 9px;
+  position: absolute;
+  top: 11px;
+  right: 11px;
+  width: 30px;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  svg {
+    display: block;
+  }
+`
+
+const CloseButton = ({ onClick }) => {
+  return (
+    <Target onClick={onClick}>
+      <CloseIcon fill="white100" />
+    </Target>
+  )
+}
+
+export interface BannerProps {
+  dismissable: boolean
+  message: string
```

Comment:
If the design spec doesn't call for it it's arguable we should support it at this time, but making this 

```
message: string | JSX.Element
```

would allow us to pass in markup to the prop.

Reply:
Hmm, interesting. On the one hand the ability to pass markup makes this component more flexible but on the other, it creates an opportunity to complicate - for instance, the spec does not specify link styles. I think I'm going to leave it for now, but we'll see how this evolves.